src/HDF5.jl

```diff
@@ -289,7 +289,7 @@ function Reference(parent::Union{File,Group,Dataset}, name::AbstractString)
     return Reference(ref[])
 end
 Base.:(==)(a::Reference, b::Reference) = a.r == b.r
-hash(x::Reference, h::UInt) = hash(x.r, h)
```

Comment:
Since `Reference` is a `struct` is this even needed? (As opposed to Datatype which is mutable)

Reply:
Oh, good point. Probably not then.